package.json

```diff
@@ -0,0 +1,154 @@
+{
+  "name": "@ipfs/public-gateway-checker",
+  "version": "1.0.0",
+  "description": "Checks which public gateways are online or not",
+  "author": "Marcin Rataj <lidel@lidel.org>",
+  "license": "Apache-2.0 OR MIT",
+  "homepage": "https://github.com/ipfs/public-gateway-checker#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ipfs/public-gateway-checker.git"
+  },
+  "bugs": {
+    "url": "https://github.com/ipfs/public-gateway-checker/issues"
+  },
+  "engines": {
+    "node": ">=16.0.0",
+    "npm": ">=7.0.0"
+  },
+  "main": "src/index.ts",
+  "files": [
+    "src",
+    "dist"
+  ],
+  "eslintConfig": {
+    "extends": "ipfs"
+  },
+  "release": {
+    "branches": [
+      "master"
+    ],
+    "plugins": [
+      [
+        "@semantic-release/commit-analyzer",
```

Comment:
Q:  are we using `@semantic-release` anywhere?

Reply:
nope. Not yet. I believe this was added by `check-aegir-project` I can remove this, but we may want to release this as an npm package eventually. See https://github.com/ipfs/public-gateway-checker/issues/192

Reply:
closing this in favor of eventually releasing as npm package, #192